app/views/coronavirus_form/transport_type.html.erb

```diff
@@ -38,5 +38,18 @@
   value: @form_responses[:transport_description]
 } %>
 
+<%= render "govuk_publishing_components/components/radio", {
+  heading: t('coronavirus_form.questions.how_much_charge.title'),
+  name: "transport_cost",
+  error_message: error_items('transport_cost'),
+  items: t('coronavirus_form.questions.how_much_charge.options').map do |_, item|
+    {
+      value: item[:label],
+      text: item[:label],
+      checked: @form_responses[:transport_cost] == item[:label],
+    }
+  end
+} %>
```

Comment:
I wonder if this could be a partial, since it's likely going to be used elsewhere?

Reply:
Aha, sorry I reviewed by commit.